pkgs/top-level/all-packages.nix

```diff
@@ -26040,4 +26040,6 @@ in
   quartus-prime-lite = callPackage ../applications/editors/quartus-prime {};
 
   go-license-detector = callPackage ../development/tools/misc/go-license-detector { };
+
+  keycloak = callPackage ../servers/keycloak {};
```

Comment:
This should be placed inside the section of `all-packages.nix` marked `### SERVERS`.

Reply:
I hope putting it between `knot-dns` and `jetty` is ok? the `### SERVERS` section looked close to, but not entirely alphabetically sorted.